etc/qgis.profile

```diff
@@ -0,0 +1,46 @@
+# Firejail profile for qgis
+# Description: GIS application
+# This file is overwritten after every install/update
+# Persistent local customizations
+include qgis.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/QtProject.conf
+
+# Allow python (blacklisted by disable-interpreters.inc)
+noblacklist ${PATH}/python3*
+noblacklist /usr/lib/python3*
+noblacklist /usr/local/lib/python3*
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-xdg.inc
+
+whitelist ${HOME}/.local/share/QGIS/
+include whitelist-common.inc
+include whitelist-var-common.inc
+
+caps.drop all
+netfilter
+nodbus
+nodvd
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+seccomp
```

Comment:
Add `protocol unix,inet,inet6`

Reply:
> Add `protocol unix,inet,inet6`

with this included in the profile QGIS was logging some errors so I decided to leave it out

Reply:
@jose1711 can you paste?

Reply:
```Critical: Could not create AF_NETLINK socket (Operation not supported)
Critical: Could not create AF_NETLINK socket (Operation not supported)
Critical: Could not create AF_NETLINK socket (Operation not supported)
Critical: Could not create AF_NETLINK socket (Operation not supported)
Critical: Could not create AF_NETLINK socket (Operation not supported)
Exception:  403 Client Error: Forbidden for url: https://www.amigocloud.com/api/v1/me/projects?summary```

the last line is related to a 3rd party plugin

Reply:
OK, then add `netlink` to the protocol list.